### DISCUSSION TITLE
Would it be possible to add the Mesh feature to a FritzBox 7390 (06.86)?

### DISCUSSION THREAD

**Discussion by Yrouel** [Freetz-NG/freetz-ng#315] [Allgemein]
Hello, I was wondering if it would be possible to backport the mesh integration to 06.86 on a 7390 (and perhaps to other models stuck on 06.86 like the 3272)

**Comment by fda77** [comment]
No, the closed-source driver kernel modules for dect,wlan, etc only work wiht the kernel version of the old firmware. So it could not be exchanged with other devices.
Btw, the things avm sells as "mesh" is not a real mesh, its only marketing... If you lose wlan connection to an ap, another is selected by all devices automatically
Sadly the 7390 is not supported by openwrt (because of missing drivers): https://openwrt.org/toh/avm/fritz.box.wlan.7390

**Reply by Yrouel** [reply]
So even if the binaries like avmnexusd would actually run there would be issues with the kernel drivers? Do you think the version in 06.9x might be similar enough to work on 06.86? Is there a way to check?

Anyway I like that feature, besides being not "real mesh", because the Fritz integrates itself perfectly in the network compared to the simpler repeater feature available in 06.86. It would be nice to have on older models to make them more useful in the house network

**Reply by fda77** [reply]
Kernel modules depend on the kernel version they was build for. AVM does not release the sources, so you cant change it
If you want it for wlan: Just set the same settings like SSID and password, and your devices will swtich if needed

**Reply by Yrouel** [reply]
Eh I tried that, it "works" but it's clunky, their new Mesh makes the whole thing smoother.
Anyway what's the criteria for the feasibility of an alien firmware instead?

**Reply by fda77** [reply]
You dont have the drivers for the hardware, avm releaeses only binary blobs
